tests/phpunit/tests/http/includeOldRequestsClass.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+/**
+ * Tests that the old Requests class is included
+ * for plugins or themes that still use it.
+ *
+ * @group http
+ */
+class Tests_HTTP_IncludeOldRequestsClass extends WP_UnitTestCase {
+
+	/**
+	 * Tests that the old Requests class is included for plugins or themes that still use it.
+	 *
+	 * @ticket 57341
+	 *
+	 * @coversNothing
+	 */
+	public function test_should_include_old_requests_class() {
+		$expected = 'The PSR-0 `Requests_...` class names in the Request library are deprecated.';
+
+		$this->expectDeprecation();
+		$this->expectDeprecationMessage( $expected );
```

Comment:
Not blocker. Let's remove variable and use direct message instead of variable.

```suggestion
		$this->expectDeprecation();
		$this->expectDeprecationMessage( 'The PSR-0 `Requests_...` class names in the Request library are deprecated.' );
```

Reply:
Removed in [8f3453f](https://github.com/WordPress/wordpress-develop/pull/3776/commits/8f3453f9587ed3fa574c98b8bb1d80cf3c4fc9b7).